book/src/slashing-protection.md

```diff
@@ -0,0 +1,134 @@
+
+* Running two validator client instances simultaneously. This could be two different
+  clients (e.g. Lighthouse and Prysm) running on the same machine, two Lighthouse instances using
+  different datadirs, or two clients on completely different machines (e.g. one on a cloud server
+  and one running locally). You are responsible for ensuring that your validator keys are never
+  running simultanously – the slashing protection DB **cannot protect you in this case**.
+* Importing keys from another client without also importing voting history.
+* If you use `--init-slashing-protection` to recreate a missing slashing protection database.
+
+## Import and Export
+
+Lighthouse supports v4 of the slashing protection interchange format described
+[here][interchange-spec]. An interchange file is a record of all blocks and attestations
+signing by a set of validator keys – basically a portable slashing protection database!
+
+You can import a `.json` interchange file from another client using this command:
+
+```bash
+lighthouse account validator slashing-protection import <my_interchange.json>
+```
+
+Instructions for exporting your existing client's database are out of scope for this document,
+please check the other client's documentation for instructions.
+
+When importing an interchange file, you still need to import the validator keystores themselves
+separately, using the instructions about [importing keystores into
+Lighthouse](./validator-import-launchpad.md).
+
+---
+
+You can export Lighthouse's database for use with another client with this command:
+
+```
+lighthouse account validator slashing-protection export <lighthouse_interchange.json>
+```
+
+[interchange-spec]: https://hackmd.io/@sproul/Bk0Y0qdGD
+
+## Troubleshooting
+
+### Misplaced Slashing Database
+
+If the slashing protection database cannot be found, it will manifest in an error like this:
+
+```
+Oct 12 14:41:26.415 CRIT Failed to start validator client        reason: Failed to open slashing protection database: SQLError("Unable to open database: Error(Some(\"unable to open database file: /home/karlm/.lighthouse/medalla/validators/slashing_protection.sqlite\"))").
+Ensure that `slashing_protection.sqlite` is in "/home/karlm/.lighthouse/medalla/validators" folder
+```
+
+Usually this indicates that during some manual intervention the slashing database has been
+misplaced. If you have correctly imported your keys, you should never see this error
+(see [Initialization](#initialization)).
+
+The safest way to remedy this error is to find your old slashing protection database and move
+it to the correct location. In our example that would be
+`~/.lighthouse/medalla/validators/slashing_protection.sqlite`. You can search for your old database
+using a tool like `find`, `fd`, or your file manager's GUI. Ask on the Lighthouse Discord if you're
+not sure.
+
+If you are absolutely 100% sure that you need to recreate the missing database, you can start
```

Comment:
I've seen this question asked on discord by a few users, and I'm not sure I know the answer.  Is there a minimum number of epochs that you can wait to recreate your slashing db to ensure that you won't get slashed? Might be worth addressing/explaining here.

Reply:
Good point. I was hesitant to say too much because I didn't want to give the wrong impression that using this flag is safe, but I think you're right it's better to give people a bit more information so they can make an informed decision. I've added that info now